client/ayon_harmony/plugins/publish/collect_instances.py

```diff
@@ -35,12 +35,13 @@ def process(self, context):
             context (:class:`pyblish.api.Context`): Context data.
 
         """
-        nodes = harmony.send(
-            {"function": "node.subNodes", "args": ["Top"]}
+        backdrops = harmony.send(
+            {"function": "Backdrop.backdrops", "args": ["Top"]} # TODO is it possible to have nested containers?
```

Comment:
@iLLiCiTiT how do we enable ruff linter for `ayon-harmony`? I'm pretty sure this comment SHOULD have come with two spaces before the `#`?